sample-config-files/omit_policies.yaml

```diff
@@ -0,0 +1,16 @@
+# YAML configuration demonstrating omitting policies from ScubaGoggles evaluation.
+# Any omitted policies should be carefully considered and documented as part of an
+# organization's cybersecurity risk management program process and practices.
+
+baselines: [gmail, chat, drive]
+
+omitpolicy:
+  GWS.GMAIL.3.1v0.3:
+    rationale: "Known false positive; our SPF policy currently cannot to be retrieved via ScubaGoggles due to a split
+      horizon setup but is available publicly."
+    expiration: "2023-12-31"
+  GWS.CHAT.5.1v0.3:
+    rationale: &DLPRationale "The DLP capability required by the baselines is implemented by third party product, [x],
```

Comment:
I like the use of use of anchors/aliases here. Perhaps for users who are unfamiliar we add a reference, https://smcleod.net/2022/11/yaml-anchors-and-aliases/ or some other url.

Reply:
Good idea, done.